src/Confluent.SchemaRegistry/Confluent.SchemaRegistry.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4;</TargetFrameworks>
```

Comment:
the previous csproj files only had net452

Reply:
`Conflient.SchemaRegistry` doesn't depend on `Apache.Avro`, so it's compatible with .net standard. we might as well target it so .NET Core people can use at least this.